.github/workflows/validate.yml

```diff
@@ -23,10 +22,67 @@ jobs:
           node-version: 12.x
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile --ignore-optional --non-interactive
+        run: yarn install --frozen-lockfile
 
       - name: Lint
         run: yarn lint
 
       - name: Test
         run: yarn test
+
+  greeter:
+    name: greeter
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@master
+        with:
+          lfs: true
+
+      - name: Set up Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+
+      - name: Test template
+        run: yarn test:template greeter
+
+  koa-rest-api:
+    name: koa-rest-api
```

Comment:
can each template use an actions matrix? (I've never used GH actions so don't really know what I'm talking about) https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
that sounds fancy so I'll check it out

Reply:
works a treat 👏